README.md

```diff
@@ -134,55 +134,89 @@ The default path is `/tmp/redhatDependencyAnalyticsReport.html`.
   
 - **Excluding dependencies with `exhortignore`**
 	<br >You can exclude a package from analysis by marking the package for exclusion.
-	If you want to ignore vulnerabilities for a dependency in a `pom.xml` file, you must add `exhortignore` as a comment against the dependency, group id, artifact id, or version scopes of that particular dependency in the manifest file.
-	For example:
+	How you exclude a package varies based on the your project's language:
 
-	```xml
-	<dependency> <!--exhortignore-->
-		<groupId>...</groupId>
-		<artifactId>...</artifactId>
-		<version>...</version>
-	</dependency>
-	```
+	- **XML**
```

Comment:
I think it would be best to write **Maven** instead of **XML** in general. As these instructions are only true for maven pom.xml files and not xml files in general.

Reply:
Good point!  I've changed it to Maven.